src/Page/Service/DefaultPageService.php

```diff
@@ -71,7 +71,9 @@ protected function updateSeoPage(PageInterface $page)
             return;
         }
 
-        if ($page->getTitle() || $page->getName()) {
+        // Do not change this line if you really know what you are doing!
+        // The page name is technically, so always prefer the title
```

Comment:
The first sentence means the opposite of what you mean, and I don't understand the second one